Bump OpenCover version

gluck opened the pull request.

gluck committed:
Bump OpenCover version

gluck committed:
bump gradle wrapper version

muryoh commented:
Looks good to me

I wanted to propose adding the proper repositories to get working builds but gradle-plugindev-plugin:1.0.4.DEV is still required as https://github.com/etiennestuder/gradle-plugindev-plugin/pull/20 was not merged

gluck commented:
btw, where can one find this 1.0.4.DEV ? may be worth uploading it to bintray so the build passes.
not sure they maintaining this plug anyway coz they now have a built-in gradle plugin mecanism, not sure we should expect an official merge (not even a release)

muryoh committed:
Fix build for online ci

- Specifies repositories
- Uploaded a custom version of the plugindev plugin
that contains a required PR

Change-Id: Ic6d5594ea64c4cffc08c1b32f7a0a71a75da822c
